streams.yml

```diff
@@ -21,37 +21,19 @@
 
 golang:
   # can be pulled from registry-proxy.engineering.redhat.com/rh-osbs/openshift-golang-builder@...
-  # https://brewweb.engineering.redhat.com/brew/buildinfo?buildID=2414523
-  # openshift-golang-builder-container-v1.19.6-202303140941.el8.g6a2861c
-  image: openshift/golang-builder@sha256:f38f31aef24699bb3dac4f22dcd5692778e612d5ea69633e937756136c8f0176
-  mirror: true
-  transform: rhel-8/golang
-  upstream_image_base: registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.19-openshift-{MAJOR}.{MINOR}.art
-  upstream_image: registry.ci.openshift.org/ocp/builder:rhel-8-golang-1.19-openshift-{MAJOR}.{MINOR}
```

Comment:
for `canonical_builders_from_upstream: on` to work we still need the previous definition to exist. i would simply change `golang` to `golang-1.19` and `golang-1.20` to `golang` (with the updated builders of course). same for `rhel-9-golang` definitions.